a11y-dialog.js

```diff
@@ -40,7 +40,7 @@
 		this._maintainFocus = this._maintainFocus.bind(this);
 		this._bindKeypress = this._bindKeypress.bind(this);
 
-		this.trigger = isString(this.options.trigger) ? getNodes(this.options.trigger, false, document, true)[0] : this.options.trigger;
+		this.trigger = isString(this.options.trigger) ? getNodes(this.options.trigger, false, document, true) : this.options.trigger;
```

Comment:
Please make second argument true, or this will fail in ie11. Also please adjust spacing to match rest of file, we arent using spaces in parens etc :)